### PR TITLE
DOC: Correct MV Normal sig

### PR DESCRIPTION
**Pull request by bashtage**
Add new kw arg to sig

### CODE REVIEW

File: numpy/random/_generator.pyx [hunk 1]
```diff
@@ -3345,7 +3345,8 @@ cdef class Generator:
     def multivariate_normal(self, mean, cov, size=None, check_valid='warn',
                             tol=1e-8, *, method='svd'):
         """
-        multivariate_normal(mean, cov, size=None, check_valid='warn', tol=1e-8)
+        multivariate_normal(mean, cov, size=None, check_valid='warn',
+                            tol=1e-8, *, method='svd')
```
Comment (charris) [inline numpy/random/_generator.pyx:3349]: Hmm. Does anyone know if it is OK to break a signature line like this? If not, we could make an exception to the 80 character limit.

**Comment by mattip** [comment]
The [docs](https://13878-908607-gh.circle-artifacts.com/0/doc/build/html/reference/random/generated/numpy.random.multivariate_normal.html) are not picking this up

**Comment by mattip** [comment]
Still no love from the [generated docs](https://13881-908607-gh.circle-artifacts.com/0/doc/build/html/reference/random/generated/numpy.random.multivariate_normal.html). Maybe there is a bug with `*` in the signature?

**Comment by bashtage** [comment]
@mattip I think that is the wrong doc string:

https://13881-908607-gh.circle-artifacts.com/0/doc/build/html/reference/random/generated/numpy.random.Generator.multivariate_normal.html?highlight=multivariate_normal#numpy.random.Generator.multivariate_normal

**Comment by bashtage** [comment]
Looks like it was ok the first time too:

https://13878-908607-gh.circle-artifacts.com/0/doc/build/html/reference/random/generated/numpy.random.Generator.multivariate_normal.html?highlight=multivariate_normal#numpy.random.Generator.multivariate_normal

**Comment by mattip** [comment]
Ahh, thanks. I wonder why I read the change as coming from `_mtrand.pyx`

**Comment by charris** [comment]
Thanks @bashtage .

**Pull request merged**